src/extension/content.js

```diff
@@ -131,6 +129,18 @@ function removeCacheParam(href = window.location.href) {
       if (sidekick) {
         // Toggle sidekick display
         sidekick.setAttribute('open', `${display}`);
+        if (!display) {
+          [...sidekick.parentElement.children].forEach((el) => {
+            if (el !== sidekick) {
+              try {
+                // @ts-ignore
+                el.style.display = 'initial';
+              } catch (e) {
+                // ignore
+              }
+            }
+          });
+        }
```

Comment:
I think this is too broad here; we should at least check if a view is active and `display: none` is set on these elements...

Reply:
added conditions , please review @rofe